<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceDialog</class>
 <widget class="QDialog" name="DeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>308</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>472</width>
    <height>308</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>472</width>
    <height>308</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New CasparCG Server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="labelDeviceIcon">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/Graphics/Images/ServerBig.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelDeviceName">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="5">
    <widget class="QLineEdit" name="lineEditDeviceName"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelAddress">
     <property name="text">
      <string>Address</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditAddress">
     <property name="placeholderText">
      <string>IP address / Hostname</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="labelPort">
     <property name="text">
      <string>Port</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEditPort">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="placeholderText">
      <string>5250</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QToolButton" name="toolButtonTestConnection">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Test Connection</string>
     </property>
     <property name="shortcut">
      <string>Alt+O</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="QLineEdit" name="lineEditDescription">
     <property name="placeholderText">
      <string>Optional</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelUsername">
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="5">
    <widget class="QLineEdit" name="lineEditUsername">
     <property name="placeholderText">
      <string>Optional</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="5">
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Optional</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QCheckBox" name="checkBoxPreview">
     <property name="text">
      <string>Preview on channel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="3">
    <widget class="QSpinBox" name="spinBoxPreviewChannel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QCheckBox" name="checkBoxLocked">
     <property name="text">
      <string>Locked on channel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="3">
    <widget class="QSpinBox" name="spinBoxLockedChannel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QCheckBox" name="checkBoxShadow">
     <property name="text">
      <string>Shadow server</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>211</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="5" colspan="2">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="shortcut">
      <string>Alt+O</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditDeviceName</tabstop>
  <tabstop>lineEditAddress</tabstop>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>lineEditUsername</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>checkBoxPreview</tabstop>
  <tabstop>spinBoxPreviewChannel</tabstop>
  <tabstop>checkBoxLocked</tabstop>
  <tabstop>spinBoxLockedChannel</tabstop>
  <tabstop>checkBoxShadow</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>DeviceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditAddress</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DeviceDialog</receiver>
   <slot>addressChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDeviceName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DeviceDialog</receiver>
   <slot>nameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonTestConnection</sender>
   <signal>clicked()</signal>
   <receiver>DeviceDialog</receiver>
   <slot>testConnection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxPreview</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DeviceDialog</receiver>
   <slot>previewChannelChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLocked</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DeviceDialog</receiver>
   <slot>lockedChannelChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addressFinished()</slot>
  <slot>nameFinished()</slot>
  <slot>testConnection()</slot>
  <slot>okClicked()</slot>
  <slot>nameChanged(QString)</slot>
  <slot>addressChanged(QString)</slot>
  <slot>previewChannelChanged(int)</slot>
  <slot>lockedChannelChanged(int)</slot>
 </slots>
</ui>
